docs/dapp/sapphire/browser.md

```diff
@@ -0,0 +1,187 @@
+---
+description: Writing Sapphire dApp for browser and Metamask
+---
+
+# Browser Support
+
+Confidential Sapphire dApps work in web browsers by wrapping the
+Metamask extension for signing and encrypting transactions.
+
+Let's begin with the [Hardhat boilerplate]. As mentioned on their website
+the boilerplate provides the following:
+
+- The Solidity contract implementing an ERC-20 token
+- Tests for the entire functionality of the contract
+- A minimal React front-end to interact with the contract using ethers.js
+
+Go ahead and clone the original [Hardhat boilerplate repo]. Move to the checked
+out folder and apply the Sapphire-specific changes to the Hardhat config file
+as described in the [quickstart].
+
+Install dependencies and deploy the contract on the Testnet:
+
+```
+pnpm install
+pnpm add -D @oasisprotocol/sapphire-paratime
+PRIVATE_KEY="0x..." pnpm hardhat run scripts/deploy.js --network sapphire
```

Comment:
I would hope that our developers are aware enough to set an actual private key, but maybe not?